src/NuGet.Core/NuGet.Frameworks/DefaultFrameworkMappings.cs

```diff
@@ -478,6 +550,8 @@ public IEnumerable<string> FrameworkPrecedence
                         FrameworkConstants.FrameworkIdentifiers.NetCore,
                         FrameworkConstants.FrameworkIdentifiers.Windows,
                         FrameworkConstants.FrameworkIdentifiers.WindowsPhoneApp,
+                        FrameworkConstants.FrameworkIdentifiers.NetStandard,
```

Comment:
I don't think this will work. Packages based platforms need to go last, this will put them ahead of all unlisted platforms.